src/vs/platform/utilityProcess/electron-main/utilityProcess.ts

```diff
@@ -237,8 +237,9 @@ export class UtilityProcess extends Disposable {
 			env['VSCODE_PARENT_PID'] = String(configuration.parentLifecycleBound);
 		}
 		if (isWindowSandboxed) {
-			env['VSCODE_CRASH_REPORTER_SANDBOXED_HINT'] = '1';
+			env['VSCODE_CRASH_REPORTER_SANDBOXED_HINT'] = '1'; // TODO@bpasero remove me once sandbox is final
```

Comment:
This is a test: https://github.com/microsoft/vscode/blob/68eb39d3e8a452422ce0a932c2192495c3b3758a/.eslintignore#L12